make/photon/db/Dockerfile.base

```diff
@@ -2,7 +2,8 @@ FROM photon:4.0
 
 ENV PGDATA /var/lib/postgresql/data
 
-RUN tdnf install -y shadow gzip postgresql13 postgresql findutils bc >> /dev/null \
+RUN tdnf install -y postgresql13 >> /dev/null
+RUN tdnf install -y shadow gzip postgresql findutils bc >> /dev/null \
```

Comment:
if this change is about installing `postgresql13` before other tools, why do we have to separate it into two `RUN tdnf install` commands, instead of using the following one: 
```
RUN tdnf install -y postgresql13 shadow gzip postgresql findutils bc >> /dev/null \
```

Reply:
`RUN tdnf install -y postgresql13 shadow gzip postgresql findutils bc >> /dev/null \`
If you install with the above, you will have this [issue](https://github.com/goharbor/harbor/issues/18672).
After my attempts to install in the following way, there will be no above [issue](https://github.com/goharbor/harbor/issues/18672).
```
RUN tdnf install -y postgresql13 >> /dev/null
RUN tdnf install -y shadow gzip postgresql findutils bc >> /dev/null \
```